rust/flatbuffers/src/builder.rs

```diff
@@ -153,7 +153,7 @@ impl<'fbb> FlatBufferBuilder<'fbb> {
         self.make_space(sz);
         {
             let (dst, rest) = (&mut self.owned_buf[self.head..]).split_at_mut(sz);
-            x.push(dst, rest);
+            unsafe { x.push(dst, rest) };
```

Comment:
how do we know this is safe?

Reply:
I'm currently working through adding SAFETY comments everywhere